packages/turbo-utils/src/getTurboConfigs.ts

```diff
@@ -0,0 +1,92 @@
+import fs from "fs";
+import path from "path";
+import getTurboRoot from "./getTurboRoot";
+import yaml from "js-yaml";
+import globby from "globby";
+import { Schema } from "turbo-types";
+import JSON5 from "json5";
+
+const ROOT_GLOB = "turbo.json";
+
+export type TurboConfigs = Array<{
+  config: Schema;
+  turboConfigPath: string;
+  workspacePath: string;
+  isRootConfig: boolean;
+}>;
+
+// A quick and dirty workspace parser
+// TODO: after @turbo/workspace-convert is merged, we can leverage those utils here
```

Comment:
Might be a good reason to make that package `@turbo/workspaces` instead of workspace-convert (single use case)